### PR TITLE
Replace Travis with GitHub Actions

### PR DESCRIPTION
**Pull request by wyster**
I think not bad idea move build to github actions, because It's totally free, without limits for open source projects

Moved all functionality from travis:
- Crontab (daily builds from latest commit in master)

- Push images from master (you can find examples on [my docker hub](https://hub.docker.com/repository/docker/wyster/php-cs-fixer/tags))
[Action report with push](https://github.com/wyster/docker-php-cs-fixer/actions/runs/419353678) (master branch temporary changed)
[Action report without push](https://github.com/wyster/docker-php-cs-fixer/actions/runs/419370404) (branch is not master)

- Push images with tag ([example on my docker hub](https://hub.docker.com/repository/docker/wyster/php-cs-fixer/tags?page=1&ordering=last_updated&name=0.3), [action report](https://github.com/wyster/docker-php-cs-fixer/actions/runs/419359588))
- Push images from release-[.0-9]+ branch ([example on my docker hub](https://hub.docker.com/repository/docker/wyster/php-cs-fixer/tags?page=1&ordering=last_updated&name=release-0.4), [action report](https://github.com/wyster/docker-php-cs-fixer/actions/runs/419353794))
- Run on pull requests to master (without push, [example](https://github.com/wyster/docker-php-cs-fixer/actions/runs/419405636), master branch temporary changed)

Latest php version was changed to 8, for the future, because now is not supported in php-cs-fixer, [issue](https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4702)

### CODE REVIEW

**Comment by cytopia** [comment]
@wyster

Thanks for the PR. I'm currently already in the process of testing different approaches on Docker builds and regular pushes to Docker hub.

### Single image builds

* Tag, Branch, PR: https://github.com/devilbox/docker-php-fpm-8.0/blob/master/.github/workflows/build.yml
* Nightly on branch and Tag: https://github.com/devilbox/docker-php-fpm-8.0/blob/master/.github/workflows/nightly.yml

### Multiple image builds

* Tag, branch, PR: https://github.com/devilbox/docker-php-fpm/blob/master/.github/workflows/build.yml
* Nightly on branch and Tag: https://github.com/devilbox/docker-php-fpm/blob/master/.github/workflows/nightly.yml

### Others

Additionally I'm also ensuring, that you never forget to update nightly.yml tags in the workflows, e.g.: https://github.com/devilbox/docker-php-fpm/blob/master/Makefile#L89

---

So basically I'm currently testing the above stated approaches and will (in case they work flawlessly) integrate them in all of the `cytopia/docker-*` projects. Thank you nonetheless for this PR and also your approach. I will have a more detailed look and see if there's anything I can copy as well :-)

**Comment by cytopia** [comment]
@wyster Has been resolved with latest release
